### PR TITLE
Rename Int to Int64 and add support for UInt64

### PR DESCRIPTION
**Pull request by OceanOak**
Changelog:

```
Area of change
- Rename Int to Int64
- Add support for unsigned 64-bit int
```

### CODE REVIEW

File: backend/src/LibExecution/RuntimeTypes.fs [hunk 1]
```diff
@@ -1200,6 +1214,7 @@ module Dval =
     | DUuid u -> Some u
     | _ -> None
 
+// Should we add asInt8, asInt16, etc?
```
Comment (OceanOak) [inline backend/src/LibExecution/RuntimeTypes.fs:1217]: Note to self: check this

File: backend/src/BuiltinCli/Libs/LanguageTools.fs
Comment (OceanOak) [inline backend/src/BuiltinCli/Libs/LanguageTools.fs:1]: I wasn't sure if those ints also should be changed to int64
https://github.com/darklang/dark/blob/5265ed4260160c6256162c9532e644face8c997c/backend/src/BuiltinCli/Libs/LanguageTools.fs#L27C1-L27C43

File: backend/tests/Tests/Serialization.TestValues.fs
Comment (OceanOak) [inline backend/tests/Tests/Serialization.TestValues.fs:1]: Note to self : check this
https://github.com/darklang/dark/pull/5150/files#diff-dc40f302623f636bcb3aaf60650ffcb0d8b82400296178e3067f9650dd6d9159R22

File: backend/src/BuiltinCli/Libs/LanguageTools.fs
Comment (pbiggar) [reply backend/src/BuiltinCli/Libs/LanguageTools.fs:1]: @StachuDotNet any thoughts? I don't know.

**Comment by pbiggar** [comment]
Looks great!

**Pull request merged**